taxi/migrations/0004_alter_car_drivers_alter_car_manufacturer.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.2.4 on 2023-08-21 16:12
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("taxi", "0003_alter_car_manufacturer"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="car",
+            name="drivers",
+            field=models.ManyToManyField(
+                related_name="cars_driven", to=settings.AUTH_USER_MODEL
+            ),
+        ),
+        migrations.AlterField(
+            model_name="car",
+            name="manufacturer",
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="cars_manufactured",
+                to="taxi.manufacturer",
+            ),
+        ),
+    ]
```

Comment:
Reduce the number of your migrations so that only one with `CreateModel` will be left. Prefer pushing one migration to make all required changes with models, there is no need to keep all of them.